package.json

```diff
@@ -267,7 +268,7 @@
     "resin-cli-visuals": "^1.8.0",
     "resin-compose-parse": "^2.1.3",
     "resin-doodles": "^0.1.1",
-    "resin-multibuild": "^4.11.0",
+    "resin-multibuild": "^4.12.1",
```

Comment:
This should be `4.12.2` to include the latest multibuild fixes and warning wording.

Reply:
Oh wait, it was already `4.12.2`. I guess my browser page was not refreshed. 🤦